docs(pages): update nodejs (#154)
neondatabase/website: content/docs/integrations/node.md

### Introduction

> In this section, you will learn how to add a Postgres client to your NodeJS project and connect to your the database.

Reviewer: ```suggestion
In this section, you will learn how to add a Postgres client to your NodeJS project and connect to your database.
```

## Step 1 - Create a NodeJS project and add dependencies

> Let's first create a NodeJS project and move to the newly created directory.

Reviewer: ```suggestion
Create a NodeJS project and change to the newly created directory.
```

```shell
mkdir neon-nodejs-example
cd neon-nodejs-example
npm init -y
```

Add project dependecies using the following command:

```shell
npm install postgres dotenv
```

## Step 2 — Store Neon credentials

> Next, store your Neon credentials in the `.env` file. Note that for security purposes, you should not expose your Neon credentials to the browser.

Reviewer: ```suggestion
Store your Neon credentials in the `.env` file. Note that for security purposes, you should not expose your Neon credentials to the browser.
```

```shell
PGHOST='<project_name>.cloud.neon.tech:<port>'
PGDATABASE='<database>'
PGUSER='<username>'
PGPASSWORD='<password>'
PROJECT_NAME='<project_name>'
```

## Step 3 — Connect to database using Postgres client and Neon credentials

> Then, to connect to the database using Postgres client and your Neon credentials, add the following code to the `src/app.js` file:

Reviewer: ```suggestion
To connect to the database using the Postgres client and your Neon credentials, add the following code to the `src/app.js` file:
```

```javascript
const postgres = require('postgres');
require('dotenv').config();

const { PGHOST, PGDATABASE, PGUSER, PGPASSWORD, PROJECT_NAME } = process.env;
const URL = `postgres://${PGUSER}:${PGPASSWORD}@${PGHOST}/${PGDATABASE}?options=project%3D${PROJECT_NAME}`;

const sql = postgres(URL, { ssl: 'require' });

async function getPostgresVersion() {
  const result = await sql`select version()`;
  console.log(result);
}

getPostgresVersion();

> ```

Reviewer 2: we can add something like:
"run `node app.js`" and enjoy the result. You can use this in your express app, for nextjs api, etc." And add links for other integrations

Author: Good idea @seymourisdead . @danieltprice , what do you think?

Reviewer 3: I think I agree with Polina's suggestion, to me this page looks a bit open-ended and missing the mention of the result of the conclusion.

Reviewer: If there is an actual result you can show, that would be great as well.

Author: Good idea @danieltprice , I can add the result too.

getPostgresVersion();
```

You can now run `node app.js` to view the result. 
```
Result(1) [
  {
    version: 'PostgreSQL 14.5 on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit'
  }
]
```

> The same configuration can be used in an Express or Next.js applications.

Reviewer: ```suggestion
The same configuration can be used in Express or Next.js applications.
```
